src/store/modules/groups.spec.js

```diff
@@ -116,7 +122,7 @@ describe('groups', () => {
     })
 
     it('can not join a group', async () => {
-      mockJoin.mockImplementationOnce(() => { throw new Error('some error') })
+      mockJoin.mockImplementationOnce(throws(Error('some error')))
```

Comment:
Ah, I didn't now `Error()` is the same as `new Error()`, looks neater, would be nice to move towards consistency. No change requested.

Reply:
It does work, but maybe it's not the best style? I remember reading that one should always use `new Error`.